### PR TITLE
Topic/casing

### PR DESCRIPTION
**Pull request by godunko**

### CODE REVIEW

File: source/text/implementation/vss-implementation-utf8_string_handlers.adb [hunk 1]
```diff
@@ -1046,6 +1047,112 @@ package body VSS.Implementation.UTF8_String_Handlers is
...
+        VSS.Implementation.UCD_Casing_UTF8.Mapping_Information :=
+          VSS.Implementation.UCD_Casing_UTF8.Mapping_Data_Table
+            (Base + Offset);
+
+      Target : UTF8_In_Place_Data
+        with Import, Convention => Ada, Address => Data'Address;
+
+   begin
+      --  Amount of the data that might be stored in in-place object is know
+      --  to be large that largest case mapping, thus all checks for this case
```
Comment (reznikmm) [inline source/text/implementation/vss-implementation-utf8_string_handlers.adb:1112]: s/large/larger/ ?

File: source/text/implementation/vss-implementation-string_handlers.ads [hunk 2]
```diff
@@ -300,4 +308,11 @@ package VSS.Implementation.String_Handlers is
    --  terminators. Line terminator (character or combination of characters)
    --  are removed unless Keep_Terminator is set to True.
 
+   not overriding procedure Get_Case_Mapping
```
Comment (reznikmm) [inline source/text/implementation/vss-implementation-string_handlers.ads:311]: why Get_Case_Mapping is not a function?

**Review by reznikmm — APPROVED** [review]

File: source/text/implementation/vss-implementation-utf8_string_handlers.adb (hunk 1 above)
Comment (godunko) [reply source/text/implementation/vss-implementation-utf8_string_handlers.adb:1112]: Fixed

File: source/text/implementation/vss-implementation-string_handlers.ads (hunk 2 above)
Comment (godunko) [reply source/text/implementation/vss-implementation-string_handlers.ads:311]: Passing of just address as parameter is faster than return of object.

**Pull request merged**